tests/countries/test_south_georgia_and_the_south_sandwich_islands.py

```diff
@@ -0,0 +1,218 @@
+        )
+
+    def test_country_aliases(self):
+        self.assertAliases(SouthGeorgiaAndTheSouthSandwichIslands, GS, SGS)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(SouthGeorgiaAndTheSouthSandwichIslands(years=2011))
+
+    def test_special_holidays(self):
+        self.assertHoliday(
+            "2022-06-02",
+            "2022-06-03",
+            "2023-05-08",
+            "2023-11-14",
+        )
+
+    def test_new_years_day(self):
+        name = "New Year's Day"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(2012, 2050)))
+        obs_dt = (
+            "2012-01-02",
+            "2017-01-02",
+            "2022-01-03",
+            "2023-01-02",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_possession_day(self):
+        name = "Possession Day"
+        self.assertHolidayName(name, (f"{year}-01-17" for year in range(2012, 2050)))
+        obs_dt = (
+            "2015-01-19",
+            "2016-01-18",
+            "2021-01-18",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_good_friday(self):
+        name = "Good Friday"
+        self.assertHolidayName(
+            name,
+            "2020-04-10",
+            "2021-04-02",
+            "2022-04-15",
+            "2023-04-07",
+            "2024-03-29",
+            "2025-04-18",
+        )
+        self.assertHolidayName(name, range(2012, 2050))
+
+    def test_easter_monday(self):
+        name = "Easter Monday"
+        self.assertHolidayName(
+            name,
+            "2012-04-09",
+            "2013-04-01",
+            "2014-04-21",
+            "2015-04-06",
+            "2016-03-28",
+            "2017-04-17",
+            "2018-04-02",
+            "2019-04-22",
+        )
+        self.assertHolidayName(name, range(2012, 2019))
+        self.assertNoHolidayName(name, range(2020, 2050))
+
+    def test_queens_birthday(self):
+        name = "The Queen's Birthday"
+        self.assertHolidayName(name, (f"{year}-04-21" for year in range(2018, 2022)))
+        obs_dt = ("2019-04-23",)
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_liberation_day(self):
+        name = "Liberation Day"
+        self.assertHolidayName(name, (f"{year}-04-26" for year in range(2012, 2016)))
+        self.assertHolidayName(name, (f"{year}-04-25" for year in range(2016, 2050)))
+        obs_dt = (
+            "2014-04-28",
+            "2015-04-27",
+            "2021-04-26",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_shackleton_day(self):
+        name = "Shackleton Day"
+        self.assertHolidayName(name, (f"{year}-05-20" for year in range(2015, 2050)))
```

Comment:
```suggestion
        self.assertHolidayName(name, (f"{year}-05-20" for year in range(2015, 2050)))
        self.assertNoHolidayName(name, range(2012, 2015))
```
And same check for similar holidays.